packages/dcos-launch/extra/launch/config.py

```diff
@@ -49,375 +21,256 @@ def expand_path(path: str, relative_dir: str) -> str:
     return os.path.abspath(os.path.join(relative_dir, path))
```

Comment:
Also, it's interesting to see much simpler (and straight-forward) config.py with cerebrus.